week-1/Homework/mandatory/2-logic-error.js

```diff
@@ -1,31 +1,40 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word;
 }
+console.log(trimWord("Code Your Future"));
 
 function getWordLength(word) {
-  return "word".length()
+  return word.length;
 }
+console.log(getWordLength("A wild sentence appeared!"));
 
 function multiply(a, b, c) {
-  a * b * c;
-  return;
+  return a * b * c;
 }
```

Comment:
Great Altom, putting the code and not return at the return keyword will give undefined.